test/failure/lexical-error.prl

```diff
@@ -1,3 +1,3 @@
-Thm BoolTest : [ (-> bool bool) true ] by [
-  { lam _. `_tt }; auto
+Thm LexicalError : [ (-> bool bool) true ] by [
+  { lam x. `_tt }; auto
```

Comment:
For some reasons `lam` do not take underscores anymore. Could it be that we do not have the "don't-care" pattern yet?